Assets/Game/Scripts/CameraController.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+
+public class CameraController : MonoBehaviour
```

Comment:
This setup always follow the character, right? I think it is good for now unless we want to have a minimap